src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -153,14 +174,42 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
         return get(config, "isExportMode", false);
     };
 
-    private createMapControls = () => {
-        this.chart.addControl(
-            new mapboxgl.NavigationControl({
-                showCompass: false,
-            }),
-            "bottom-right",
-        );
+    private createMapControls = (): void => {
+        const {
+            config: { isDisabledInteractive },
+        } = this.props;
+
+        if (!isDisabledInteractive) {
+            this.chart.addControl(this.navigationControl, "bottom-right");
+            return;
+        }
+
+        if (this.chart.loaded()) {
+            this.chart.removeControl(this.navigationControl);
+        }
```

Comment:
Your code makes me confused at the first time :)
I suggest to reformat like this

```
if (isDisabledInteractive) {
        if (this.chart.loaded()) {
            this.chart.removeControl(this.navigationControl);
        }
} else {
        this.chart.addControl(this.navigationControl, "bottom-right");
}
```

What could happen if 
- we remove `navigationControl ` which is not added before?
- at this time, chart is not loaded?

Reply:
It is the same result but I prefer to keep
```
        if (!isDisabledInteractive || !isExportMode) {
            this.chart.addControl(this.navigationControlButton, "bottom-right");
            return;
        }

        if (this.chart.loaded()) {
            this.chart.removeControl(this.navigationControlButton);
        }
```

so that have not nested if